docs/4.x/tests-github

```diff
@@ -0,0 +1,31 @@
+---
+category: Develop
+previous: tests-js-tracker
+next: tests-troubleshooting
+---
+# Running tests/builds on GitHub Action
+
+[Github Action](https://github.com/features/actions) is a continuous integration tool that will run tests/builds for a GitHub repository every time commits are pushed.
+
+## Matomo on GitHub Action 
+
+Matomo uses GitHub Action to automatically run its build on every commit (for every branch and pull request).
+
+Each developer is responsible to keep the build green.
+
+## Github Action Scripts
+
+All the Github Action build file are locate in `.github/workflows/*.yml`
+
+Each scripts are trigger by its own conditions, for more details see [Github Action Docs](https://docs.github.com/en/actions)
+
+## Create a new CI script
+
+To create a new CI script please read [Quickstart for GitHub Actions](https://docs.github.com/en/actions/quickstart) 
+
+We recommend fork the Matomo project first and run on your own pipeline first. Also, please check the list before get starts.
+
+
+- Marketplace: To use a marketplace action/container, please always choose the official service or container.
```

Comment:
Guess it might be good to explain that a bit more in detail. We should also mention that used actions should always be pinned to a full length commit SHA (unless they are published by github). We could maybe add some examples what to do and what to avoid

Also when using third-party action it's necessary to do a small audit of the action (in the used version), to ensure it doesn't do anything bad.